src/overlays/ovl_i1/sf_ve1.c

```diff
@@ -154,17 +333,31 @@ void func_i1_80192CB0(Actor* actor) {
 
 #pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_80192E2C.s")
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_80192EA4.s")
+void func_i1_80192EA4(Actor* arg0) {
+}
 
 #pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_80192EB0.s")
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_801933B4.s")
+void func_i1_801933B4(Actor* arg0) {
+    arg0->scale = -1.0f;
+    arg0->obj.pos.y += 488.0f;
+}
 
 #pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_801933DC.s")
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_801934D0.s")
+void func_i1_801934D0(Actor* arg0) {
+    Matrix_Translate(gGfxMatrix, 0.0f, -488.0f, 0.0f, 1U);
+    Animation_DrawSkeleton(0, &D_900D164, arg0->vwork, NULL, NULL, arg0, &gIdentityMatrix);
+}
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_80193540.s")
+void func_i1_80193540(Object_80* arg0) {
+    f32 temp_fv1;
+
+    temp_fv1 = arg0->obj.pos.z;
```

Comment:
temp_fv1 looks fake, can you try removing it?

Reply:
Yeah, it's fake. removed on this commit "e31fd0a"